src/server/session.ts

```diff
@@ -714,6 +714,9 @@ module ts.server {
                     case CommandNames.Quickinfo: {
                         var quickinfoArgs = <protocol.FileLocationRequestArgs>request.arguments;
                         response = this.getQuickInfo(quickinfoArgs.line, quickinfoArgs.col, quickinfoArgs.file);
+                        if (!response) {

```

Comment:
i do not understand how this is diffrent from just throwing an error in getQuickInfo?